django_rest/django_rest/views.py

```diff
@@ -0,0 +1,98 @@
+from django.shortcuts import get_object_or_404
+from django.contrib.auth.hashers import make_password
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from django_rest.models import Employee
+from django.contrib.auth.models import User
+from django_rest.serializers import EmployeeGetSerializer, EmployeeSerializer
+from rest_framework.permissions import IsAuthenticated, IsAdminUser
+from drf_yasg import openapi
+from drf_yasg.utils import swagger_auto_schema
+
+
```

Comment:
This functionality should be in the [next story](https://mahonya.atlassian.net/browse/PRA-10).